job-descriptions/ux-designer.md

```diff
@@ -0,0 +1,78 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# UX Designer (San Francisco or remote)
+
+### About us
+
+Sourcegraph's mission is to enable every software developer to create products using the best technology. The innovations of the future will all rely on software, and we want to bring the future sooner by building the best tools that help developers answer questions about their code. You can learn more about our mission by reading [our master plan](https://sourcegraph.com/plan).
+
+Our values:
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we need to have a diverse team. Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are searching for a creative, self-motivated UX designer who can iteratively improve the cohesive user expereience, while bringing clean and thoughtful UI to their designs. It is important that this person deeply understands developers and can craft an experience that delights a technical audience.
```

Comment:
> It is important that this person deeply understands developers

Is this a pre-requisite to applying?

Reply:
Adjusted role description and added this to the qualifications instead.